src/bikeracks/templates/base.html

```diff
@@ -118,13 +118,61 @@
                       </div>
                       
                   </li>
+                  <li class="nav-item">
+                    <button class="btn btn-secondary" data-toggle="modal" data-target="#guideModal">Guide</button>
+                  </li>
                   <li class="nav-item">
                       <button class="btn btn-secondary" data-toggle="modal" data-target="#feedbackModal">Contact</button>
                   </li>
                 
               </ul> 
           </div> <!-- /.navbar-collapse -->
       </nav> 
+
+      <!-- Guide Modal -->
+
+      <div id="guideModal" class="modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <p>How to Use</p>
+              <button id="closeModal" type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <h6>Everyone can:</h6>
```

Comment:
This section reads like a specification of the behavior rather than a guide to help users.

Imagine the least technical person you know trying to use this site. Does this text help them? Writing good docs is hard. Writing good user-facing docs is even harder.

Try being more prescriptive, e.g. "You can log in using your Google account in the upper right."